classes/article/ArticleDAO.inc.php

```diff
@@ -182,11 +183,14 @@ function updateObject($article) {
 		);
 
 		$this->updateLocaleFields($article);
+		$contextId = Request::getContext()->getId();
```

Comment:
It is always better to try not to use Request statically -- whenever possible. So maybe to get the context id from the article object here?